biome.json

```diff
@@ -0,0 +1,59 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.5.3/schema.json",
+  "organizeImports": {
+    "enabled": true
+  },
+  "linter": {
+    "enabled": true,
+    "rules": {
+      "recommended": true,
+      "suspicious": {
```

Comment:
Just to check if I understood the intention correctly, everything is turned off so that we introduce Biome with minimal initial changes to the codebase?

Reply:
yes ser